x-pack/plugins/uptime/server/lib/requests/search/enrich_monitor_groups.ts

```diff
@@ -294,6 +294,11 @@ const getHistogramForMonitors = async (
       query: {
         bool: {
           filter: [
+            {
+              range: {
+                'summary.down': { gt: 0 },
+              },
+            },
```

Comment:
That's great optimisation. I would have never though something like this.

Reply:
Does it also needs to use exists filter on summary?

Reply:
Nope, it will just ignore anything without that field